### PR TITLE
HAWQ-1380. Keep hawq_toolkit schema check in HAWQ native side

### PR DESCRIPTION
**Pull request by interma**
very similar to the handle way of information_schema

### CODE REVIEW

**Comment by interma** [comment]
@linwen @stanlyxiang @zhangh43 help to review, thanks!

File: src/backend/catalog/aclchk.c [hunk 1]
```diff
@@ -2685,8 +2685,12 @@ List *getActionName(AclMode mask)
...
 bool checkNamespaceFallback(Oid x)
 {
-  if (x == PG_CATALOG_NAMESPACE || x == information_schema_namespcace_oid
-     || x == PG_AOSEGMENT_NAMESPACE || x == PG_TOAST_NAMESPACE || x == PG_BITMAPINDEX_NAMESPACE)
+  if (x == PG_CATALOG_NAMESPACE
+     || x == PG_AOSEGMENT_NAMESPACE
+     || x == PG_TOAST_NAMESPACE
+     || x == PG_BITMAPINDEX_NAMESPACE
+     || x == information_schema_namespcace_oid
+     || x == hawq_toolkit_schema_namespcace_oid )
```
Comment (linwen) [inline src/backend/catalog/aclchk.c]: one typo, namespace.

Comment (interma) [inline src/backend/catalog/aclchk.c]: Thanks! Fixed.

**Comment by interma** [comment]
1. added feature-test of HAWQ-1380
2. fixed HAWQ-1381 coredump

@linwen @zhangh43 @wcl14 @stanlyxiang help to review, tks!

File: src/test/feature/Ranger/test_ranger.cpp [hunk 2]
```diff
@@ -157,3 +154,59 @@ TEST_F(TestHawqRanger, BasicTest) {
...
+TEST_F(TestHawqRanger, FallbackTest) {
+    SQLUtility util;
+
+    if (util.getGUCValue("hawq_acl_type") == "ranger")
+    {
+		string rootPath(util.getTestRootPath());
+		string rangerHost = RANGER_HOST;
+		string initfile = "Ranger/sql/init_file";
+		string cmd;
+		int FallbackCase = 10000;
```
Comment (interma) [inline src/test/feature/Ranger/test_ranger.cpp]: I assumed < 10000 is generated sql, >=10000 is manual writed sql.

**Comment by interma** [comment]
merged

**Pull request closed without merging**